feat(third-party-contrib): update entries (#20)
honeycombio/third-party-contrib: README.md

| honeycomb-logger | Nginx proxy that logs to stdout events sent to Honeycomb. | https://github.com/darklang/honeycomb-logger |
| cl-honeycomb | Common Lisp client for sending events | https://github.com/franzinc/cl-honeycomb/ |
| terraform-provider-honeycombio | Terraform provider for managing Honeycomb resources | https://github.com/kvrhdn/terraform-provider-honeycombio 

> | gha-buildevents | Add observability to your GitHub Actions workflows | https://github.com/kvrhdn/gha-buildevents

Reviewer: ```suggestion
| gha-buildevents | Trace GitHub Action workflows and send to Honeycomb | https://github.com/kvrhdn/gha-buildevents
```

Reviewer 2: LGTM. I agree that this is more appropriate 👍🏻
